packages/utils/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@edx/frontend-enterprise-utils",
+  "version": "0.0.0",
+  "description": "Utils and other miscellaneous enterprise things.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/edx/frontend-enterprise.git"
+  },
+  "main": "src/index.js",
+  "browserslist": [
+    "last 2 versions",
+    "ie 11"
+  ],
+  "scripts": {
+    "build": "make build",
+    "i18n_extract": "BABEL_ENV=i18n fedx-scripts babel src --quiet > /dev/null",
+    "lint": "fedx-scripts eslint --ext .js --ext .jsx .",
+    "lint:fix": "fedx-scripts eslint --fix --ext .js --ext .jsx .",
+    "snapshot": "fedx-scripts jest --updateSnapshot",
+    "start": "fedx-scripts webpack-dev-server --progress",
+    "test": "fedx-scripts jest --coverage --passWithNoTests"
+  },
+  "author": "edX",
+  "license": "AGPL-3.0",
+  "homepage": "https://github.com/edx/frontend-enterprise#readme",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/edx/frontend-enterprise/issues"
+  },
+  "sideEffects": false,
+  "dependencies": {
+    "@testing-library/react": "11.2.6",
```

Comment:
should this be dep or devDep?

Reply:
It is currently a regular dependency and after some consideration, I left it as a regular dependency due to that the `renderWithRouter` test utility function we have in the `frontend-enterprise-utils` package relies on `@testing-library/react` as an implementation detail, so to ensure it gets bundled when publishing to NPM, it will need to remain a dependency.